src/cpp/include/openvino/genai/streamer_base.hpp

```diff
@@ -18,6 +18,7 @@ class OPENVINO_GENAI_EXPORTS StreamerBase {
     /// @brief put is called every time new token is decoded,
     /// @return bool flag to indicate whether generation should be stopped, if return true generation stops
     virtual bool put(int64_t token) = 0;
+    virtual bool put(const std::vector<int64_t>& tokens) = 0;
```

Comment:
Explain in doc-strings what the new overload is for.

Reply:
Doc-string added